modules/profile/files/channelmgnt.json

```diff
@@ -173,4 +173,15 @@
          "Universal_Omega"
       ]
    }
+    "#miraheze-feed-test":{
+      "chanops":[
+         "dmehus",
+         "Reception123",
+         "PuppyKun",
```

Comment:
This should be his account name not his nick

Reply:
@MacFan4000 Oh, do you mean PuppyKun? What's his account name? I thought it was PuppyKun. heh

Reply:
NDKilla

Reply:
@MacFan4000 Oh, okay, thanks. I'm surprised I never noticed that. Fixing now.